src/forms.css

```diff
@@ -42,10 +42,14 @@
 
 .input:focus,
 .textarea:focus,
-.select:focus {
+.select.select--stroke:focus {
   box-shadow: inset 0 0 0 1px var(--default-form-color);
 }
 
+.select:focus {
+  box-shadow: none;
+}
```

Comment:
I think we can cut this rule.

Reply:
Without this rule, the default focus styling is applied (shown below). I didn't think this looked great due to the lack of left padding -- what do you think? 

<img width="772" alt="Screen Shot 2021-04-14 at 2 19 40 PM" src="https://user-images.githubusercontent.com/15935667/114780609-7dd27b00-9d2c-11eb-8d55-a9404798946a.png">

Reply:
Aha, the more prominent tab style, even though it's ugly, might actually be the best move. Right now tab active state for selects is disorienting because it's so subtle.

Thinking about this more, what if we did this: write an even higher priority tab state selector that used `[data-assembly-focus-control='visible']`, and that applied _two box shadows_: one that is a single stroke, and a second box shadow that matches the tab active states we use elsewhere in assemby (for links, buttons, ect).

Reply:
> Right now tab active state for selects is disorienting because it's so subtle.

That's true, just changing the text + arrow color to be slightly darker isn't noticeable unless you know to look for it. 

> what if we did this: write an even higher priority tab state selector that used [data-assembly-focus-control='visible'], and that applied two box shadows: one that is a single stroke, and a second box shadow that matches the tab active states we use elsewhere in assemby (for links, buttons, ect).

This makes a lot of sense to me! Maybe with one caveat -- for stroked buttons, the focus styling actually uses just one box-shadow and removes the thinner, colored box-shadow applied via the relevant color modifier. What do you think about using the same approach for selects?

<img width="570" alt="Screen Shot 2021-04-14 at 2 57 23 PM" src="https://user-images.githubusercontent.com/15935667/114785615-ce000c00-9d31-11eb-90c3-107e0718df96.png">
<img width="574" alt="Screen Shot 2021-04-14 at 2 57 14 PM" src="https://user-images.githubusercontent.com/15935667/114785616-ce98a280-9d31-11eb-9fc6-919a3d84c033.png">


This does break consistency with how colored textareas handle focus, though. This would go beyond the scope of this PR, but maybe the correct approach is to update both selects and _also_ textareas to have two box-shadows when focused? To help illustrate, the yellow textarea here has a tab active state, but it's very subtle.

<img width="998" alt="Screen Shot 2021-04-14 at 2 55 08 PM" src="https://user-images.githubusercontent.com/15935667/114785425-86798000-9d31-11eb-8451-5ab0015921bf.png">

Reply:
@adrianababakanian ooh thanks for documenting this. We should ticket making the focus state for textareas and text inputs more prominent, maybe also with the double box-shadow technique?

Reply:
I went ahead with adding a double box-shadow for focused stroked selects in this PR per your suggestion @samanpwbb , I'll go ahead and open a ticket for improving focus states for textareas and text inputs too 👍